metalwihen/WordPress-Android
# Add analytics events to Prompts Screen

Fixes #17125 

This PR introduces analytics event tracking to the Prompts List Screen.

**To test:**
- Follow test instructions as explained in previous PRs: #2, #5, #6
- Open the Prompts Screen to see if event `blogging_prompts_list_accessed` is tracked
- Click on the different tabs of the Prompts Screen to see if event `blogging_prompts_list_tab_changed` is tracked.

**Screenshots**
Attaching screenshots of the logs that confirm that the events are getting logged:

![image](https://user-images.githubusercontent.com/2140539/195433186-35023725-c8a0-4383-aa4e-73e80f3f80c5.png)
![image](https://user-images.githubusercontent.com/2140539/195433212-562c55ca-74fd-4291-ad16-7e52960ee9a2.png)


## Regression Notes
1. Potential unintended areas of impact
- Changes were made only to the Prompts Screen 

2. What I did to test those areas of impact (or what existing automated tests I relied on)
- Run existing unit tests
- Run manual tests around the Prompts screen to see if flow is working as expected

3. What automated tests I added (or what prevented me from doing so)
- Add unit tests to test AnalyticsTracker and ViewModel

PR submission checklist:
